nni/tools/nnictl/launcher.py

```diff
@@ -11,6 +11,7 @@
 
 from nni.experiment import Experiment, RunMode
 from nni.experiment.config import ExperimentConfig, convert, utils
+from nni.nas.experiment.pytorch import RetiariiExperiment
```

Comment:
This will make pytorch a "required" dependency of NNI. We should avoid this.

Reply:
good point, let me put it into the functions